lib/queue.js

```diff
@@ -284,11 +293,11 @@ Queue.prototype.isReady = function(){
 }
 
 Queue.prototype.getJobMoveCount = function(){
-  return this.bclient.commandQueue.length;
+  return this.client.commandQueue.length;
 };
 
 Queue.prototype.whenCurrentMoveFinished = function(){
```

Comment:
What was this for originally? Seems like it can be removed to reduce complexity.